help/en/html/doc/Technical/Patterns.shtml

```diff
@@ -33,6 +33,38 @@ structure and patterns by looking at older code pieces.
 This page attempts to describe the recommended structure and patterns,
 and point to examples of current best practices.
 
+<a id="namedbeans" name="namedbeans"></a><h2>Names, NamedBeans, and Managers</h2>
```

Comment:
>> +<a id="namedbeans" name="namedbeans"></a><h2>Names, NamedBeans, and Managers</h2>
>
> You might want to point out that these /are not/ JavaBeans.

I think that would be very helpful. I had assumed that they were, or 
were at least somehow related.

steve